backend/src/main/kotlin/app/ehrenamtskarte/backend/stores/database/repos/AcceptingStoresRepository.kt

```diff
@@ -76,6 +84,57 @@ object AcceptingStoresRepository {
             .limit(limit, offset)
     }
 
+    fun determineRemovableAcceptingStoreId(acceptingStore: AcceptingStore, project: ProjectEntity): Int? {
+        return AcceptingStores.innerJoin(PhysicalStores).innerJoin(Addresses).innerJoin(Contacts)
+            .slice(AcceptingStores.id).select {
+                (Addresses.street eq acceptingStore.streetWithHouseNumber) and
+                    (Addresses.postalCode eq acceptingStore.postalCode!!) and
+                    (Addresses.location eq acceptingStore.location) and
+                    (Addresses.countryCode eq acceptingStore.countryCode) and
+                    (Contacts.email eq acceptingStore.email) and
+                    (Contacts.telephone eq acceptingStore.telephone) and
+                    (Contacts.website eq acceptingStore.website) and
+                    (AcceptingStores.name eq acceptingStore.name) and
+                    (AcceptingStores.description eq acceptingStore.discount) and
+                    (AcceptingStores.categoryId eq acceptingStore.categoryId) and
+                    (AcceptingStores.regionId.isNull()) and // TODO #538: For now the region is always null
+                    (AcceptingStores.projectId eq project.id) and
+                    (
+                        PhysicalStores.coordinates eq Point(
+                            acceptingStore.longitude!!,
+                            acceptingStore.latitude!!
+                        )
+                        )
+            }.firstOrNull()?.let { it[AcceptingStores.id].value }
+    }
+
+    fun createStore(acceptingStore: AcceptingStore, project: ProjectEntity) {
```

Comment:
optional and nit-picky: if we only use projectId here (and above too), I think we could just pass projectId as a param